user/TASK/chassis_task/chassis_task.h

```diff
@@ -56,16 +69,21 @@ typedef struct
     const motor_measure_t *motor_feedback;
     
     //Current speed read from motors
-    int16_t pos_read;
+    uint16_t pos_read;
```

Comment:
There should be some documentation somewhere on Slite or something that explains why this is unsigned while the rest are signed